crates/iota-protocol-config/src/lib.rs

```diff
@@ -1263,6 +1268,10 @@ impl ProtocolConfig {
         self.feature_flags.passkey_auth
     }
 
+    pub fn iota_metadata_module_bytes(&self) -> bool {
```

Comment:
Can we keep the same order for the getters, as for the definition of the protocol parameters?
IMO we should move this getter after `pub fn consensus_commit_transactions_only_for_traversed_headers(&self) -> bool {`